modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
-<suite name="Identity-suite-initializer" parallel="false" thread-count="1">
+<suite name="Identity-suite-initializer" parallel="false" thread-count="1" preserve-order="true">
```

Comment:
As discussed offline, ` preserve-order="true"` lets remove this, if its not required.